package.json

```diff
@@ -99,14 +99,15 @@
         "@fortawesome/fontawesome-svg-core": "^1.2.34",
         "@fortawesome/free-solid-svg-icons": "^5.15.2",
         "@fortawesome/react-fontawesome": "^0.1.14",
-        "@tweakpane/core": "^1.0.6",
+        "@tweakpane/core": "^1.0.8",
         "@tweakpane/plugin-essentials": "^0.1.4",
+        "acorn": "^8.7.0",
```

Comment:
I was getting some inexplicable `npm install` warnings about acorn needing to be installed. I'm not really sure what acorn is for - I am going to look into this now.

Reply:
yes, or peer... I think it's really supposed to be peer

Reply:
i reset everything and was able to get rid of the acorn dep.

Reply:
Awesome!